Code/GraphMol/ChemReactions/DaylightParser.cpp

```diff
@@ -141,6 +152,19 @@ ChemicalReaction *RxnSmartsToChemicalReaction(
   std::vector<std::string> productSmarts =
       DaylightParserUtils::splitSmartsIntoComponents(productText);
 
+  // if the input includes CX extensions, they will show up here after the
+  // last product. We can't currently deal with those, so issue a warning and
+  // strip out anything following the last productSmarts
+  // This was Github #4759
```

Comment:
The comment indicates a warning is issued but the code doesn’t issue a warning.

Reply:
doh!
Thanks for noticing that.

Reply:
@bp-kelley : that's now fixed